keep/README.md

```diff
@@ -0,0 +1,57 @@
+# Agent Check: Keep
+
+## Overview
+
+Keep is an open-source AIOps alert management tool.
+
+Keep acts as a single pane of glass and helps you turn 1000s into just 10s of meaningful alerts.
+
+We use monitors data, logs and events to correlate alerts and reduce noise.
+
+## Setup
+
+### Installation
+
+The Keep integration is installed using the (Integration Tile)[https://app.datadoghq.com/integrations/keephq] on your Datadog account, with OAuth2.
+
+Users will:
+1. Be redirected to Keep's platform
+2. Click on the Datadog tile to initate an OAuth flow
+3. Review and confirm the required scopes in their Datadog account
```

Comment:
@emarsha94 I hope this makes sense, I couldn't find a great example somewhere else so I'm pretty much in the dark here 😆 

```suggestion
1. Clicking on Keep's integration tile -> Install Integration
2. User will be redirected to Keep's platform for sign-in
3. User will be redirected back to Datadog to review and confirm the required scopes from their Datadog account
4. User will be redirected back to Keep's platform to see whether installation was successfully completed
```